### PR TITLE
Organize README.md

### PR DESCRIPTION
**Pull request by fcolecumberri**

### CODE REVIEW

**Review by port19x — APPROVED** [review]
I like it. Categorizing the tools makes a lot of sense

**Comment by johnalanwoods** [comment]
@yozachar - good with you?

File: README.md [hunk 1]
```diff
@@ -1,46 +1,76 @@
 <h1 align="center">Modern Unix</h1>
 
+
+
+
+<h2 align="center">Manuals/Docs Utils</h2>
```
Comment (yozachar) [inline README.md:6]: Keep sub headings to the left.

```md
## Manual/Doc Utils
```

**Review by yozachar — CHANGES_REQUESTED** [review]
Looks good.

File: README.md [hunk 2]
```diff
@@ -1,46 +1,76 @@
...
+| :--------------------------: | ------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------- | :---------------------------------------------------------------------------- |
+|       [`cheat`][cheat]       | Create and view interactive cheatsheets on the command-line.                                                       | ![_][cheat-ghst] ![_][cheat-lang] ![_][cheat-ghlc]                   | <img alt="cheat-preview" width="300" src="./screenshots/cheat.png" />         |
+|        [`tldr`][tldr]        | A community effort to simplify `man` pages with practical examples.                                                | ![_][tldr-ghst] ![_][tldr-lang] ![_][tldr-ghlc]                      | <img alt="tldr-preview" width="300" src="./screenshots/tldr.png" />           |
+
+<h2 align="center">System Monitor</h2>
```
Comment (yozachar) [inline README.md:13]:
```md
## System Monitor Uitls
```

File: README.md [hunk 3]
```diff
@@ -1,46 +1,76 @@
...
-|       [`cheat`][cheat]       | Create and view interactive cheatsheets on the command-line.                                                       | ![_][cheat-ghst] ![_][cheat-lang] ![_][cheat-ghlc]                   | <img alt="cheat-preview" width="300" src="./screenshots/cheat.png" />         |
+|     [`glances`][glances]     | A `top`/`htop` alternative for most operating systems.                                                             | ![_][glances-ghst] ![_][glances-lang] ![_][glances-ghlc]             | <img alt="glances-preview" width="300" src="./screenshots/glances.png" />     |
+|        [`gtop`][gtop]        | System monitoring dashboard for terminal.                                                                          | ![_][gtop-ghst] ![_][gtop-lang] ![_][gtop-ghlc]                      | <img alt="gtop-preview" width="300" src="./screenshots/gtop.gif" />           |
+
+<h2 align="center">Unix Utils Clones/Improvements</h2>
```
Comment (yozachar) [inline README.md:22]:
```md
### GNU/*Nix Clone Utils
```

File: README.md [hunk 4]
```diff
@@ -1,46 +1,76 @@
...
 |      [`trashy`][trashy]      | a cli system trash manager, alternative to rm and trash-cli                                                        | ![_][trashy-ghst] ![_][trashy-lang] ![_][trashy-ghlc]                | <img alt="trashy-preview" width="300" src="./screenshots/trashy.gif" />       |
-|          [`xh`][xh]          | A friendly and fast tool for sending HTTP requests.                                                                | ![_][xh-ghst] ![_][xh-lang] ![_][xh-ghlc]                            | <img alt="xh-preview" width="300" src="./screenshots/xh.gif" />               |
 |      [`zoxide`][zoxide]      | A smarter `cd` command inspired by `z`.                                                                            | ![_][zoxide-ghst] ![_][zoxide-lang] ![_][zoxide-ghlc]                | <img alt="zoxide-preview" width="300" src="./screenshots/zoxide.riff" />      |
 
+<h2 align="center">Network/Web Related</h2>
```
Comment (yozachar) [inline README.md:46]:
```md
## Network Utils
```

File: README.md [hunk 5]
```diff
@@ -1,46 +1,76 @@
...
+|         [`mtr`][mtr]         | A network diagnostic tool, combining functionality of `traceroute` and `ping`                                      | ![_][mtr-ghst] ![_][mtr-lang] ![_][mtr-ghlc]                         | <img alt="mtr-preview" width="300" src="./screenshots/mtr.png" />             |
+|    [`rustscan`][rustscan]    | The Modern Port Scanner. Find ports quickly (3 seconds at its fastest). `nmap` alternative.                        | ![_][rustscan-ghst] ![_][rustscan-lang] ![_][rustscan-ghlc]          | <img alt="rustscan-preview" width="300" src="./screenshots/rustscan.gif" />   |
+|          [`xh`][xh]          | A friendly and fast tool for sending HTTP requests.                                                                | ![_][xh-ghst] ![_][xh-lang] ![_][xh-ghlc]                            | <img alt="xh-preview" width="300" src="./screenshots/xh.gif" />               |
+
+<h2 align="center">Git/Diff</h2>
```
Comment (yozachar) [inline README.md:58]:
```md
## Git/Diff Utils
```

File: README.md [hunk 6]
```diff
@@ -1,46 +1,76 @@
...
+|       [`delta`][delta]       | A viewer for `git` and `diff` output.                                                                              | ![_][delta-ghst] ![_][delta-lang] ![_][delta-ghlc]                   | <img alt="delta-preview" width="300" src="./screenshots/delta.png" />         |
+|   [`diffutils`][diffutils]   | Drop-in replacement of [`diffutils`][gnu-diffutils] in Rust (by [`uutils`][uutils])                                | ![_][diffutils-ghst] ![_][diffutils-lang] ![_][diffutils-ghlc]       | NA                                                                            |
+|     [`lazygit`][lazygit]     | simple terminal UI for git commands                                                                                | ![_][lazygit-ghst] ![_][lazygit-lang] ![_][lazygit-ghlc]             | <img alt="lazygit-preview" width="300" src="./screenshots/lazygit.gif" />     |
+
+<h2 align="center">Misc</h2>
```
Comment (yozachar) [inline README.md:66]:
```md
## Misc. Utils
```

**Review by yozachar — CHANGES_REQUESTED** [review]

**Comment by fcolecumberri** [comment]
Changes done.

**Comment by johnalanwoods** [comment]
Merged and thank you!

**Pull request merged**